src/Controller/EnMarche/CitizenInitiativeController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche;
+
+use AppBundle\CitizenInitiative\CitizenInitiativeCommand;
+use AppBundle\Controller\EntityControllerTrait;
+use AppBundle\Entity\Skill;
+use AppBundle\Form\CitizenInitiativeType;
+use AppBundle\Repository\SkillRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/initiative_citoyenne")
+ */
+class CitizenInitiativeController extends Controller
+{
+    use EntityControllerTrait;
+
+    /**
+     * @Route("/creer", name="app_create_citizen_initiative")
+     * @Method("GET|POST")
+     * @Security("is_granted('CREATE_CITIZEN_INITIATIVE')")
+     */
+    public function createEventAction(Request $request, ?CitizenInitiativeCommand $command): Response
+    {
+        $command = new CitizenInitiativeCommand($this->getUser());
+        $form = $this->createForm(CitizenInitiativeType::class, $command);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->get('app.citizen_initiative.handler')->handle($command);
+            $this->addFlash('info', $this->get('translator')->trans('citizen_initiative.creation.success'));
```

Comment:
Flash messages should be translated in the flash template and they are https://github.com/EnMarche/en-marche.fr/blob/master/templates/base.html.twig#L94.